platform/linux-generic/arch/aarch64/odp_sysinfo_parse.c

```diff
@@ -202,6 +202,49 @@ int cpuinfo_parser(FILE *file, system_info_t *sysinfo)
 
 void sys_info_print_arch(void)
 {
+	int value;
+
+	printf("ARM FEATURES:\n");
+	printf("  __ARM_ARCH           %i\n",  __ARM_ARCH);
+
```

Comment:
This is failing on Shippable:
```
  CC       arch/aarch64/odp_sysinfo_parse.lo
arch/aarch64/odp_sysinfo_parse.c: In function ‘sys_info_print_arch’:
arch/aarch64/odp_sysinfo_parse.c:207:41: error: ‘__ARM_ARCH’ undeclared (first use in this function)
  printf("  __ARM_ARCH           %i\n",  __ARM_ARCH);
                                         ^
arch/aarch64/odp_sysinfo_parse.c:207:41: note: each undeclared identifier is reported only once for each function it appears in
Makefile:1217: recipe for target 'arch/aarch64/odp_sysinfo_parse.lo' failed
make[1]: *** [arch/aarch64/odp_sysinfo_parse.lo] Error 1
```

Reply:
Shippable has quite old GCC version: 4.8.5. Surprisingly it does not define ARM_ARCH. I'll ifdef it also. Should we update shippable SW level ?

Reply:
@psavol Shippable has available update. I clicked to install it. Will see if it will update gcc or not.

Reply:
@psavol @muvarov this is interesting. I could not find used gcc in the docker image mounts.

Reply:
./configure output has to print it

Reply:
Hmmm. @muvarov who controls `odp/dev` image? It is used for Shippable builds. Most probably we should switch to our images built from `Linaro/odp-docker-images`?

Reply:
@muvarov could you please enable shippable build for odp-docker-image with empty .shippable.yml file for now? I'd try building new aarch64 image from that repo.

Reply:
@lumag enabled in Shippable web interface